src/game/common/ini.cpp

```diff
@@ -142,6 +139,8 @@ int INIClass::Save(FileClass &file) const
 
 int INIClass::Save(Pipe &pipe) const
 {
+    static const char *_newline = "\r\n";
```

Comment:
Should be static `const char _newline[] = "\r\n";` to be more efficient.

Reply:
Noted, changed.